docker-compose.dev.yml

```diff
@@ -49,12 +49,13 @@ services:
     environment:
       - DATASTORE_READONLY_PASSWORD=${DATASTORE_READONLY_PASSWORD}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - CKAN_DB_PASSWORD=${CKAN_DB_PASSWORD}
```

Comment:
Style note here -- if you're not renaming the environment variables, it can just be:
```
environment:
  - CKAN_DB_PASSWORD
```

In that case, it will pull in the identically named environment variable.